### PR TITLE
Update + Optimize Statistics

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/Statistics.cs [hunk 1]
```diff
@@ -26,6 +25,9 @@ public sealed partial class Statistics : PositionAndSizeForm
 {1}
 ============================= Most Used Lines =============================
 {2}";
+
+        private static readonly char[] ExpectedChars = new[] { '♪', '(', ')', '[', ']', ' ', ',', '!', '?', '.', ':', ';', '-', '_', '@', '<', '>', '/', '0', '1', '2', '3', '4', '5', '6', '7', '8', '9' };
```
Comment (ivandrofly) [inline src/Forms/Statistics.cs]: Also added `(, ), [, ], ♪`

File: src/Forms/Statistics.cs [hunk 2]
```diff
@@ -162,125 +158,111 @@ private void Statistics_KeyDown(object sender, KeyEventArgs e)
...
+            foreach (KeyValuePair<string, int> item in hashtable)
             {
-                if (int.Parse(item.Value) > 1)
+                if (item.Value > 1)
                 {
-                    string s = item.Value;
-                    while (s.Length < 4)
-                        s = "0" + s;
-                    sortedTable.Add(s + "_" + item.Key, item.Value + ": " + item.Key);
+                    sortedTable.Add($"{item.Value:0000}" + "_" + item.Key, item.Value + ": " + item.Key);
```
Comment (ivandrofly) [inline src/Forms/Statistics.cs]: `($"{item.Value:0000}"` is one of the new feature of C# 6.0

**Comment by ivandrofly** [comment]
The new c# feat is what 'cause AppVeyor to fails :wink:

**Comment by niksedk** [comment]
Perhaps we should switch to VS2015.
I guess we probably need to update some project/config and other files

**Comment by ivandrofly** [comment]
> Perhaps we should switch to VS2015.
> I guess we probably need to update some project/config and other files

yeah, there are some cool new feat introduced with it :)

File: src/Forms/Statistics.cs [hunk 3]
```diff
@@ -162,125 +158,111 @@ private void Statistics_KeyDown(object sender, KeyEventArgs e)
...
                 }
                 else if (s.Length > 0 && s.Contains(' '))
                 {
-                    hashtable.Add(s, "1");
+                    hashtable.Add(s, 1);
                 }
             }
         }
 
+        private static string StripHtmlTags(string text)
```
Comment (niksedk) [inline src/Forms/Statistics.cs]: perhaps use HtmlUtil.RemoveHtmlTags ?
(I also wonder if AppVoyor will be okay if you change anything now...)

Comment (xylographe) [inline src/Forms/Statistics.cs]: Yes, [AppVeyor now uses **Microsoft (R) Build Engine version 14.0.24720.0**](https://ci.appveyor.com/project/SubtitleEdit/subtitleedit/build/3780) to build SE (previously was 12.0.*).

Comment (ivandrofly) [inline src/Forms/Statistics.cs]: > perhaps use HtmlUtil.RemoveHtmlTags ?

Ah nope, `StripHtmlTags` holds different logic as ` HtmlUtil.RemoveHtmlTags`

**Pull request merged**